doc/api/net.markdown

```diff
@@ -51,7 +51,8 @@ var server = net.createServer((socket) => {
 });
 
 // grab a random port.
-server.listen(() => {
+server.listen((err) => {
+  if(err) throw err;
```

Comment:
Do whatever the linter is fine with and what's most consistent in that file. We have a rule in place for the missing space after `if`, but do not require things like the [`curly`](http://eslint.org/docs/rules/curly.html) rule yet.